mgmt_cli_test.py

```diff
@@ -195,6 +196,45 @@ def verify_backup_success(self, mgr_cluster, backup_task, keyspace_name='keyspac
         self.restore_backup_from_backup_task(mgr_cluster=mgr_cluster, backup_task=backup_task,
                                              keyspace_and_table_list=per_keyspace_tables_dict)
 
+    def restore_schema_with_task(self, mgr_cluster, backup_task, timeout):
+        snapshot_tag = backup_task.get_snapshot_tag()
+        restore_task = mgr_cluster.create_restore_task(restore_schema=True, location_list=self.locations,
+                                                       snapshot_tag=snapshot_tag)
+        restore_task.wait_and_get_final_status(step=30, timeout=timeout)
+        assert restore_task.status == TaskStatus.DONE, f"Schema restoration of {snapshot_tag} has failed!"
+        self.db_cluster.restart_scylla()  # After schema restoration, you should restart the nodes
+
+    def restore_data_with_task(self, mgr_cluster, backup_task, timeout):
+        snapshot_tag = backup_task.get_snapshot_tag()
+        restore_task = mgr_cluster.create_restore_task(restore_data=True, location_list=self.locations,
+                                                       snapshot_tag=snapshot_tag)
+        restore_task.wait_and_get_final_status(step=30, timeout=timeout)
+        assert restore_task.status == TaskStatus.DONE, f"Schema restoration of {snapshot_tag} has failed!"
+        for node in self.db_cluster.nodes:
+            node.run_nodetool("repair")  # After data restoration, you should repair every node
+
+    def run_verification_read_stress(self):
```

Comment:
-  It's not used what's the purpose of it?
- most of it is a copy-paste from longevity test and is not used. Why not to extract to separate method?
- Name indicates 'run' which is usually just running, not waiting for finish - thus name is misleading

Reply:
* For future PRs
* Correct. I separated it to a method. Please take a look.
* I see. Should I rename it to `run_and_verify_verification_read_stress`?

Reply:
Please, do not add unused code.
Unused code requires maintenance for no gain.

Reply:
It will be used by another PR (that is already open: https://github.com/scylladb/scylla-cluster-tests/pull/5834).
I just pushed this part in advance since there are multiple tasks I'm working on ATM that already use this code

Reply:
Ok.